### PR TITLE
Feature/ui update on refactor

### PR DESCRIPTION
**Pull request by wyyxdgm**

### CODE REVIEW

File: src/files/.openlink.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/files/OATH-Whitepaper-CN.pdf
```
Comment (hwwang-oath) [inline src/files/.openlink.txt]: Soft link

```
ln -snf <new-whitepager-en.pdf> OATH-One-Pager-EN.pdf
```

File: src/files/OATH-One-Pager-CN.pdf [hunk 2]
```diff
@@ -1 +1 @@
-oath-one-pager-v2.3-zh_CN.pdf
```
Comment (hwwang-oath) [inline src/files/OATH-One-Pager-CN.pdf]: This should be a soft link

File: src/files/OATH-One-Pager-EN.pdf [hunk 3]
```diff
@@ -1 +1 @@
-oath-one-pager-v2.4-en_US.pdf
\ No newline at end of file
+one-pager_2.6.0_en.pdf
```
Comment (hwwang-oath) [inline src/files/OATH-One-Pager-EN.pdf]: This should be a soft link

File: src/files/OATH-Whitepaper-CN.pdf [hunk 4]
```diff
@@ -0,0 +1 @@
+Oath_Whitepaper_2.4-zh_CN.pdf
```
Comment (hwwang-oath) [inline src/files/OATH-Whitepaper-CN.pdf]: This should be a soft link

File: src/files/OATH-Whitepaper-EN.pdf [hunk 5]
```diff
@@ -0,0 +1 @@
+Oath_whitepaper_2.4-en_US.pdf
```
Comment (hwwang-oath) [inline src/files/OATH-Whitepaper-EN.pdf]: This should be a soft link

File: src/files/.openlink.txt (hunk 1 above)
Comment (wyyxdgm) [reply src/files/.openlink.txt]: 这个文件只是记录用

**Pull request merged**